scram-tools.file/tools/rocm/env.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/bash -e
 export COMPILER_HOST=$(gcc -dumpmachine)
+export HSA_XNACK=1
```

Comment:
@fwyzard , looks like this is not used any more. Env set here is used by toolfile generation script which replaced `@ENV_VAR@`  in the toolfile to `$ENV_VAR`.

Reply:
Ah... how do I make `cmsenv` export `HSA_XNACK=1` ?

Reply:
for cmsenv, you just need to add `<runtime name="HSA_XNACK" value="1"/>` in the `rocm.xml` toolfile. If you want it to be configurable then use `<runtime name="HSA_XNACK" value="@HSA_XNACK@"/>` in the toolfile and add some  login the `scram-tools.file/tools/rocm/env.sh` to set `HSA_XNACK` env ( e.g. by checking the `$TOOL_ROOT/lib` directory for hsa libs? )

Reply:
OK, thanks.
No, it doesn't need to be configurable.